tests/apollo/util/skvbc.py

```diff
@@ -181,7 +181,8 @@ def max_key(cls):
             checkpoints_num=2,
             persistency_enabled=True):
         initial_nodes = self.bft_network.all_replicas(without=stale_nodes)
-        [self.bft_network.start_replica(i) for i in initial_nodes]
+        self.bft_network.start_all_replicas()
+        self.bft_network.stop_replicas(stale_nodes)
```

Comment:
Makes sense. This is because in the containerized environment the nodes are already running

Reply:
Yup!